### DISCUSSION TITLE
Link from public to basic

### DISCUSSION THREAD

**Discussion by renrom** [Budibase/budibase#10416] [Help] [answered]
This issue: https://github.com/Budibase/budibase/discussions/8234 , is that still an open issue? Are the plans to let it work as described? so some sort of public welcome or even some app screens and some screens where authentication is needed?

**Comment by melohagan** [comment, marked as answer]
Hey @renrom 

The issue has been inactive for a while. I've moved the issue into our triage queue and we will discuss it with the developers next week!

**Reply by renrom** [reply]
Thanks!
